capstone/src/test/java/com/google/univiz/CarnegieSizeDegreeTest.java

```diff
@@ -0,0 +1,20 @@
+package com.google.univiz;
+
+import static com.google.common.truth.Truth.assertThat;
+
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.junit.runners.JUnit4;
+
+@RunWith(JUnit4.class)
+public final class CarnegieSizeDegreeTest {
+
+  @Test
+  public void testIndexing() {
+    assertThat(CarnegieSizeDegree.NOT_AVAILABLE).isEqualTo(CarnegieSizeDegree.getDegree(0));
+    assertThat(CarnegieSizeDegree.VSMALL_NONRESI_FOUR_YEAR_COLLEGE)
+        .isEqualTo(CarnegieSizeDegree.getDegree(6));
+    assertThat(CarnegieSizeDegree.SMALL_TWO_YEAR_COLLEGE)
+        .isEqualTo(CarnegieSizeDegree.getDegree(2));
```

Comment:
Just wanted to make a note that typically asserts are written like:
`assertThat(actual).isEqualtTo(expected)`.

You'll get a lot more support from the fluent assertions like this, especially when you get into lists:
`assertThat(actual).contains("foo")`